### PR TITLE
[MONDRIAN-2155] - Invalid results when using native filter and topcount with aggregate table measures

### PR DESCRIPTION
**Pull request by akhayrutdinov**
- use aggregation table's rollup aggregator instead of measure's
- add tests

@mkambol, @lucboudreau, review it please

### CODE REVIEW

File: testsrc/main/mondrian/rolap/NativeFilterAgainstAggTableTest.java [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+            + "Axis #2:\n"
+            + "{[Measures].[Sales Count]}\n"
+            + "Row #0: 1,959\n"
+            + "Row #0: 4,090\n";
+
+        doTestFilteringOnAggregatedBy("COUNT", query, expectedResult);
+    }
+
+    public void testFilteringOnAggregated_ByDistinctCount() {
+        String query = ""
```
Comment (mkambol) [inline testsrc/main/mondrian/rolap/NativeFilterAgainstAggTableTest.java]: This test does not actually use an aggregate table for the native SqlTupleReader query.

**Comment by akhayrutdinov** [comment]
@mkambol, I removed that case and updated the PR

**Comment by pamval** [comment]
@mkambol please merge (if acceptaple) at your earliest convenience. We're finalizing the October SP today and tomorrow

**Comment by lucboudreau** [comment]
Pulled locally. Ran all tests. No regressions. Code looks good.

**Pull request merged**